bin/tessel-2.js

```diff
@@ -186,12 +189,24 @@ parser.command('init')
 
 parser.command('wifi')
   .callback(function(opts) {
-    //TODO: Refactor switch case into controller.wifi
+    // TODO: Refactor switch case into controller.wifi
     if (opts.list) {
       controller.printAvailableNetworks(opts)
         .then(closeSuccessfulCommand, closeFailedCommand);
-    } else if (opts.ssid && opts.password) {
-      controller.connectToNetwork(opts)
+    } else if (opts.ssid || opts.password) {
+      if (opts.ssid && opts.password) {
```

Comment:
If we move the SSID and Password check here (which makes sense so you don't have to wait for it to connect to a Tessel) we might as well remove the check that already exists here: https://github.com/tessel/t2-cli/pull/282/files#diff-ad1c3af214118b233a97e4dd4f67d729R127